modules/pruning-deployments.adoc

```diff
@@ -2,48 +2,66 @@
 //
 // * applications/pruning-objects.adoc
 
+:_content-type: PROCEDURE
 [id="pruning-deployments_{context}"]
-= Pruning DeploymentConfig objects
+= Pruning deployment resources
 
-To prune `DeploymentConfig` objects that are no longer required by the system due to age and status, administrators can run the following command:
+You can prune resources associated with deployments that are no longer required by the system, due to age and status.
+
+The following command prunes replication controllers associated with `DeploymentConfig` objects:
 
 [source,terminal]
 ----
 $ oc adm prune deployments [<options>]
 ----
 
-.Prune deployments CLI configuration options
-[cols="4,8",options="header"]
+[NOTE]
+====
+To also prune replica sets associated with `Deployment` objects, use the `--replica-sets` flag. This flag is currently a Technology Preview feature.
```

Comment:
```suggestion
To prune replica sets associated with `Deployment` objects, use the `--replica-sets` flag. This flag is currently a Technology Preview feature.
```
The "also" didn't seem necessary here since this is a NOTE.

Reply:
Per slack convo, leaving in the "also" since it means "in addition to pruning replication controllers"